.travis/run-in-docker

```diff
@@ -17,8 +25,12 @@ run_in_container ()
     if ${as_root-false}; then
         uid=0
     fi
-
+if [ "$ARCH" = "ppc64le" ]; then
+    docker exec -u $uid -i "$container_id" "$@"
+else
```

Comment:
Why we can not use `-t` here, too?

Reply:
I removed 't' from here because it was throwing error like "The input device is not a TTY". 

Thanks for review and comment.

Reply:
Ok, it deserves a comment above the line ... and perhaps we could drop `-t` for all architectures?

Reply:
Yes, we can remove it and i have done that as well. Thanks !!